api/nodes/views.py

```diff
@@ -40,7 +40,7 @@ def get_default_odm_query(self):
         return (
             Q('is_public', 'eq', True) &
             Q('is_deleted', 'ne', True) &
-            Q('is_dashboard', 'ne', True)
```

Comment:
I realize now that this implementation is correct if we want to exclude folders/dashboard. +1